src/plugins/data/common/search/search_source/normalize_sort_request.ts

```diff
@@ -49,6 +49,10 @@ function normalize(
     }
   }
 
+  // FIXME: for unknown reason on the server this setting is serialized
```

Comment:
can we create an issue for this and link it here ?

Reply:
https://github.com/elastic/kibana/issues/89902